lib/resty/chash.lua

```diff
@@ -58,6 +55,12 @@ local function load_shared_lib(so_name)
 
     local cpath = package.cpath
 
+    local postfix = ".so"
+    if ffi.os == "OSX" then
+        postfix = ".dylib"
+    end
+    so_name = so_name .. postfix
```

Comment:
I remember that when the ext name is missing, LuaJIT will automatically add it according to the platform. Therefore, we don't need to add it by ourselves.

Reply:
Maybe we can add macOS as a new test environment for this change? https://docs.travis-ci.com/user/reference/osx/

Reply:
thank u, i remove the ext name.